by-language/java-jooq/backlog.rst

```diff
@@ -0,0 +1,66 @@
+###################################################
+Backlog for Java jOOQ demo application with CrateDB
+###################################################
+
+
+Features
+========
+
+- Evaluate more features of jOOQ, like:
+
+  - `Using JDBC batch operations`_
+  - `jOOQ for PROs`_, specifically but not exclusively `exporting`_ and `importing`_
+  - `Alternative execution models`_
+  - `The static query DSL API`_
+  - `Configuration and setup of the generator`_
+
+- jOOQ's code generator does not know about CrateDB's special data types.
+  Evaluate if this will work when using the dynamic access variant, without
+  using code generation. Of course, the container data types ``OBJECT`` and
+  ``ARRAY`` are the most interesting ones.
+
+
+Other topics
+============
+
+- When jOOQ connects to CrateDB, it displays ``SET SESSION STATEMENT WILL BE
+  IGNORED: extra_float_digits`` on the server console.
```

Comment:
Do you think this observation should be carried over to the CrateDB repository in form of an issue?